src/main/java/com/ibm/wala/examples/analysisscope/analysisscope.java

```diff
@@ -0,0 +1,36 @@
+package com.ibm.wala.examples.analysisscope;
+
+import com.ibm.wala.ipa.callgraph.AnalysisScope;
+import com.ibm.wala.ipa.callgraph.AnalysisScope;
+import com.ibm.wala.util.config.FileOfClasses;
+import com.ibm.wala.util.config.AnalysisScopeReader;
+
+import java.io.File;
+import java.io.IOException;
+import com.ibm.wala.util.config.AnalysisScopeReader;
+
+//for more information, please check out https://github.com/wala/WALA/wiki/Analysis-Scope
+
+
+public class analysisscope {
```

Comment:
```suggestion
public class AnalysisScopeExamples {
```
Class names start with a capital and use camel case.  See https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names